### PR TITLE
MC-12212: Add docs for creates

### PR DESCRIPTION
**Pull request by Gerudhoh**
#### Changes made

- Added the create documents for Ingress and Service entities

#### Related PRs
- [ksdk](https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/144)
- [k8s-plugin](https://github.com/cloudops/cloudmc-kubernetes-plugin/pull/41)

### CODE REVIEW

File: source/includes/kubernetes/_k8_ingresses.md [hunk 1]
```diff
@@ -156,3 +156,67 @@ Retrieve an ingress and all its info in a given [environment](#administration-en
...
+| ------------------------------------------ | ------------------------------------------------------------------------|
+| `apiVersion` <br/> _string_                | The api version (versioned schema) of the ingress.                      |
+| `metadata` <br/>_object_                   | The metadata of the ingress.                                            |
+| `metadata.name` <br/>_string_              | The name of the ingress.                                                |
+| `spec`<br/>_object_                        | The specification used to create and run the ingress.                   |
+| `spec.rules`<br/>_object_                  | The list of host rules used to configure the ingress.                   |
+
+| Optional Attributes                        | &nbsp;                                                                  |
+| ------------------------------------------ | ------------------------------------------------------------------------|
+| `kind`<br/>_string_                        | The string value representing the REST resource this object represents. |
```
Comment (jasondias9) [inline source/includes/kubernetes/_k8_ingresses.md]: resource **of** (or for?) this object (same comment in the other places `kind` is described)

**Review by jasondias9 — APPROVED** [review]
1 minor typo - aside from that LGTM ✅

File: source/includes/kubernetes/_k8_ingresses.md (hunk 1 above)
Comment (Gerudhoh) [reply source/includes/kubernetes/_k8_ingresses.md]: Omg good catch! I've fixed it here(ed9f4683d05136bfb0f14fd19bcca5ab0dfd5d5d) for just the ingress and service but it's a problem in the other k8s docs. I'll fix those in another PR ;)

Comment (jasondias9) [reply source/includes/kubernetes/_k8_ingresses.md]: I figured 😛 Thanks!!

**Review by Shabirmean — APPROVED** [review]

**Pull request merged**